dap_prinz_green_jobs/pipeline/green_measures/green_measures.py

```diff
@@ -188,34 +191,3 @@ def get_industry_measures(self, job_advert: Dict[str, str]) -> List[dict]:
         )
 
         return ind_green_measures_list
-
-    def get_green_measures(
-        self, job_advert: Dict[str, str], skill_list: Optional[List[str]] = None
-    ) -> Dict[str, List[dict]]:
```

Comment:
The reason I took it out was because we weren't using it, so it felt like we'd have to maintain this function if things changed unneccessarily. But it's true that it could be useful at some point and it's not super complex anyway - so I'll add it back in!